src/UI/Movies/MoviesCollection.js

```diff
@@ -257,6 +257,6 @@ Collection = AsFilteredCollection.call(Collection);
 Collection = AsSortedCollection.call(Collection);
 Collection = AsPersistedStateCollection.call(Collection);
 
-var data = ApiData.get('movie?page=1&pageSize='+pageSize+'&sortKey=sortTitle&sortDir=asc');
-
-module.exports = new Collection(data.records, { full : false, state : { totalRecords : data.totalRecords} }).bindSignalR();
+var data = ApiData.get('movie?page=1&pageSize=100000&sortKey=sortTitle&sortDir=asc');
```

Comment:
The only issue I see here, is that the ui now has to wait for all movies to be loaded. It would be nice if only the first page would be loaded initially and then then immediately after that a moviecollection.fullcollection.fetch() would be called. This way the ui loads much faster.